README.md

```diff
@@ -162,7 +162,7 @@ Provide parameters to the stack via `cdk deploy --parameters pcluster:KEY=VALUE`
 
  * `cdk ls`          list all stacks in the app
  * `cdk synth`       emits the synthesized CloudFormation template
- * `cdk deploy`      deploy this stack to your default AWS account/region
+ * `cdk deploy`      deploy this stack to your default AWS account/region (remember to update the version number on the quick launch buttons in the README!)
```

Comment:
Why is this comment here? Is there a better way we can document this? Like a release checklist?

Reply:
That is a good idea. I used this section of the README the same as a release checklist. Will return to this in a future commit.